### PR TITLE
hermitize

### PR DESCRIPTION
**Pull request by jiyoontbd**
```shell
hermit init .
hermit install openjdk@11
hermit install gradle@8.2 # current version per running ./gradlew --version
```

replaced gradlew wrapper validator action with init hermit step to publish ci and docs yaml

### CODE REVIEW

**Review by KendallWeihe — APPROVED** [review]
nice! pinging the hermit himself @alecthomas anything stand out?

only other thing which comes to my mind is maybe adding relevant developer instructions to the README

**Comment by alecthomas** [comment]
Ah yes, one thing: I would suggest not using channels. I'd pin the versions exactly then either manually upgrade them, or use Renovate to manage them - it has Hermit support.

**Comment by alecthomas** [comment]
Also, you'll need to add this to your `build.gradle.kts` file:

```
tasks.findByName("wrapper")?.enabled = false
```

That will prevent Gradle from trying to download the wrapper automatically.

**Comment by michaelneale** [comment]
oh this would be nice - then I don't have to mess around with JVM versions by hand?

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -30,7 +32,7 @@ jobs:
           key: gradle-${{ runner.os }}-${{ hashFiles('**/*.gradle.kts') }}-${{ hashFiles('**/gradle/wrapper/gradle-wrapper.properties') }}-${{ hashFiles('**/versions.properties') }}
 
       - name: Run Gradle Tasks
-        run: ./gradlew build koverXmlReport
```
Comment (alecthomas) [inline .github/workflows/ci.yaml:33]: FYI it's not necessary to include `bin/` here - `activate-hermit` will add `bin` to the `$PATH` for you.

Comment (jiyoontbd) [reply .github/workflows/ci.yaml:33]: awesome!

File: README.md [hunk 2]
```diff
@@ -273,8 +274,6 @@ If you've already cloned the repo without `--recurse-submodules`, you can do the
 git submodule update --init
 ```
 
-copying the schemas and test vectors into the protocol package's `resources` directory can be done by running `./gradlew syncSchemas` and `./gradlew syncTestVectors` respectively.
```
Comment (KendallWeihe) [inline README.md:276]: is this no longer relevant?

Comment (jiyoontbd) [reply README.md:276]: correct @KendallWeihe - schemas and test vectors are now coming in via `tbdex` submodule, so all we have to do is the command above to keep in sync, or push changes if we make changes to the schema or test vectors here.

**Review by mistermoe — APPROVED** [review]

**Review by phoebe-lew — APPROVED** [review]

**Pull request merged**